sdk/resourcemanager/Azure.ResourceManager.Core/src/OperationsBase.cs

```diff
@@ -91,6 +91,16 @@ private ProviderContainer GetProviderContainer()
         /// <returns> A valid Azure resource type. </returns>
         protected abstract ResourceType ValidResourceType { get; }
 
+        private TagOperations _tagOperations;
+
+        /// <summary>
+        /// Gets the TagOperations.
+        /// </summary>
+        protected internal TagOperations GetTagOperations()
+        {
+            return _tagOperations ??= new TagOperations(this, Id);
```

Comment:
We can make this a property since we are only newing it up once

protected internal TagOperations TagOperations => _tagOperations ??= new TagOperations(this, Id);

Reply:
Done